doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -394,8 +394,11 @@ Requests to the environment
       reference ::= @qualid
       | @string {? % @scope_key }
 
-   Displays the full name of objects from Coq's various qualified namespaces such as terms,
-   modules and Ltac, thereby showing the module they are defined in.  It also displays notation definitions.
+   Displays the full name of objects from Coq's various qualified namespaces
+   such as terms, modules and Ltac, thereby showing the module they are defined
+   in.  It also displays notation definitions.  Note that the command only reports
+   information about modules that have been loaded, such as through a
+   :cmd:`Require Import` command.
```

Comment:
I guess Import is not needed for references but is needed for notations?

Reply:
```suggestion
   :cmd:`Require` or a :cmd:`Require Import` command.
```

I guess applying the suggestion above would make things less precise but more accurate. Note that the terminology "loading" a module is usually associated with the `Require` command.

Reply:
It could also be `Require Export`.  The doc uses "loading" for all three commands.  The "such as" was to indicate that there may be other ways (e.g. I didn't want to mention `prelude.v`).  How about if I just mention "Require"?

Reply:
> I guess Import is not needed for references but is needed for notations?

Do you have a specific example in mind?  Note that `prelude.v` loads a number of modules.

Reply:
One can also load modules directly on the command-line using options which act the same as `Require` / `Require Import` / `Require Export`. The prelude could be considered as being loaded that way.

>Do you have a specific example in mind?

```coq
Module foo.
  Notation " [ x ] " := (S x).
  Definition bar := 0.
End foo.

Locate "[ _ ]". (* Unknown notation *)
Locate bar. (* Constant Top.foo.bar (shorter name to refer to it in current context is foo.bar) *)
Import foo.
Locate "[ _ ]". (* Notation " [ x ] " := (S x) (default interpretation) *)
Locate bar. (* Constant Top.foo.bar *)
```

> How about if I just mention "Require"?

Given the behavior for notations, and given that `Locate` is most often used to work with notations, if only one command is mentioned, it should be `Require Import` like you did.